cmd/root.go

```diff
@@ -79,16 +85,18 @@ func init() {
 	RootCmd.PersistentFlags().BoolVar(&noRemediations, "noremediations", false, "Disable printing of remediations section")
 	RootCmd.PersistentFlags().BoolVar(&jsonFmt, "json", false, "Prints the results as JSON")
 	RootCmd.PersistentFlags().BoolVar(&pgSQL, "pgsql", false, "Save the results to PostgreSQL")
+	RootCmd.PersistentFlags().BoolVar(&filterOpts.Scored, "scored", false, "Run only scored CIS checks")
+	RootCmd.PersistentFlags().BoolVar(&filterOpts.Unscored, "unscored", false, "Run only unscored CIS checks")
```

Comment:
I'm not convinced by the naming / semantics of these options. If you were to set both `scored` and `unscored` flags to true, then that would appear to be saying "run only scored checks AND run only unscored checks" which is paradoxical! I haven't checked what the logic in the code actually does but perhaps a better way of phrasing this would be: 

- `scored` defaults to true, and means `run the scored checks`
- `unscored` defaults to true, and means `run the unscored checks`

If you specify neither of the flags, all the checks get run (backwards compatible)
If you specify either of the flags to be true, that makes no difference as it matches the default
if you specify one of the flags to be false, that set of checks is not run (so if you specify both the flags to be false, none of the checks are run). 

Wdyt?

Reply:
Thanks for pointing this out @lizrice . The logic in the code is indeed paradoxical. I really like the semantics you proposed. Let me implement it. Should be quick.

Reply:
Should work as specified now